app/views/dvr/cannotRaiseChallenge.scala.html

```diff
@@ -16,23 +16,38 @@
 
 @import models.CanChallengeResponse
 @import config.ApplicationConfig
+@import  utils.Formatters
 
-@this(mainLayout: views.html.mainLayout)
+@this(mainLayout: views.html.mainLayout, govukButton: GovukButton)
 
-@(model: CanChallengeResponse, homePageUrl: String, authorisationId: Long, backLinkUrl: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+@(model: CanChallengeResponse, address: String, localAuth: String, homePageUrl: String, authorisationId: Long, backLinkUrl: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
 
 @mainLayout(
     pageTitle = messages("cannotRaiseChallenge.title"),
     backUri = Some(backLinkUrl)
 ) {
+    <ul class="govuk-list">
```

Comment:
2 things here:
- don't we have this assessment header already in some Twirl template?
- English text should be pulled out of `messages` file, not hard-baked into Twirl template

Reply:
we have this assessment header in check frontend, and it looks like this

```html

<div id="assessment-heading" class="govuk-grid-column-two-thirds">
    <ul class="govuk-list">
        <li id="billingAuthorityReference">
            <span class="govuk-body-s govuk-!-font-weight-bold">@Messages("assessment_heading.billingAuthorityReference"):  </span>
            <span class="govuk-body-s">@assessment.billingAuthorityCode/@assessment.billingAuthorityReferenceNumber</span>
        </li>
        <li id="address">
            <span class="govuk-body-s govuk-!-font-weight-bold">@Messages("assessment_heading.property"): </span>
            <span class="govuk-body-s">@assessment.fullAddress</span>
        </li>
    </ul>
</div>
```

it uses `span` with govuk css classes instead of `<b>` elements

Reply:
@PeterPerhac  in PL theres only this used else where   <ul class="govuk-list">
        <li>
            <b>Local authority reference:</b> @model.localAuthorityReference
        </li>
        <li>
            <b>Property:</b> @Formatters.capitalisedAddress(model.address)
        </li>
    </ul>

Reply:
we should not be using the `<b>` tag, instead we should be using a span with the appropriate govuk CSS classes (like we do in check).

about the b tag:
> This was formerly known as the Boldface element, and most browsers still draw the text in boldface. However, you should not use <b> for styling text; instead, you should use the CSS font-weight property to create boldface text, or the <strong> element to indicate that text is of special importance.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b